src/Laravel/config.php

```diff
@@ -87,6 +87,14 @@
 	//'pipeline_dir' => 'min',
 
 	/**
+	 * Which files extensions should be excluded from minification
+	 * They will be just concatenated
+	 *
+	 * @var array
+	 */
+	//'exclude_minification_regex' => '/(.min.js|-min.js)$/i';
```

Comment:
You are right. Yours pattern even significantly faster:

Time 1:  4.6014785766602E-5
Time 2:  1.8835067749023E-5